packages/atomic/src/components/atomic-folded-result-list/atomic-folded-result-list.stories.tsx

```diff
@@ -0,0 +1,402 @@
+import {html} from 'lit-html';
+import defaultStory from '../../../.storybook/default-story';
+
+const {defaultModuleExport, exportedStory} = defaultStory(
+  'Atomic/ResultList',
+  'atomic-folded-result-list',
+  {},
+  {
+    engineConfig: {
+      search: {
+        preprocessSearchResponseMiddleware: (response) => {
+          response.body.results[0] = resultWithChildren;
+          return response;
+        },
+      },
+    },
+    additionalChildMarkup: () => html`
+      <atomic-result-template>
+        <template>
+          <atomic-result-link></atomic-result-link>
+          <div style="padding-left: 15px; margin-top: 20px;">
```

Comment:
yes! I have that component already, but because we are waiting for Johnny's designs, it doesn't do anything... I'll make sure I update storybook when I do them. will add to the jira ticket